src/uu/split/src/split.rs

```diff
@@ -125,28 +127,50 @@ pub fn uu_app() -> Command {
         )
         .arg(
             Arg::new(OPT_NUMERIC_SUFFIXES)
-                .short('d')
                 .long(OPT_NUMERIC_SUFFIXES)
+                .require_equals(true)
                 .default_missing_value("0")
                 .num_args(0..=1)
+                .overrides_with(OPT_NUMERIC_SUFFIXES)
```

Comment:
Each argument should also override with the corresponding option right? Maybe we could also add some tests for that.

Reply:
yep, tests are actually covering that, but will refactor with .overrides_with_all() instead